dotnet/src/SemanticKernel/CoreSkills/TextSkill.cs

```diff
@@ -94,6 +94,21 @@ public string Uppercase(string text)
     [SKFunction("Convert a string to lowercase.")]
     public string Lowercase(string text)
     {
-        return text.ToLower(System.Globalization.CultureInfo.CurrentCulture);
+        return text.ToLowerInvariant();
```

Comment:
I think these should stay the CurrentCulture, thoughts?

Reply:
I moved those to be Invariant after a comment from Shawn on the Length being CurrentCulture, but this is giving more alerts than I would expect, will keep all as current culture.